src/common/migration/dashboards.rs

```diff
@@ -143,3 +143,54 @@ pub async fn run() -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+macro_rules! _get_variables {
+    ($map:ident, $dash:ident) => {
+        if let Some(vars) = &$dash.variables {
+            for v in vars.list.iter() {
+                if let Some(ref qd) = v.query_data {
+                    $map.entry((qd.stream.clone(), qd.stream_type))
+                        .or_default()
+                        .push(qd.field.clone());
+                }
+            }
+        }
+    };
+}
+
+fn get_query_variables(
+    dashboard: Option<&Dashboard>,
+) -> HashMap<(String, StreamType), Vec<String>> {
```

Comment:
Moving this macro and function here since this is now the only place these are used and, other than containing these definitions, the `/service/db/dashboards` module doesn't really need to exist anymore to the extent that it served as an interface between application logic in the `service` module and the `meta` table in the DB.